greenboot-health-check.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [ -x /usr/libexec/yggdrasil/device-worker ]; then
```

Comment:
WDYT of adding a check for the yggdrasil service ?

Reply:
Sure - I'm working on it : https://issues.redhat.com/browse/ECOPROJECT-340